### PR TITLE
Redux usage

### PR DESCRIPTION
**Pull request by lutangar**
> enhance #96 

- use hooks composition to fetch configurations
- remove unused app reducer actions
- create dedicated kepler reducer file
- store instance configuration in app reducer

### CODE REVIEW

**Comment by lutangar** [comment]
@OlivPi @Pipazoul here's my preliminary work on #97 still a draft tho, but I'm curious :
- the `app` reducer was sometimes populated with *stuff* but none of these was actually used right ? I didn't see any selector trying to get something from the `state.app` slice...
- the main idea is to do the heavy lifting on the keplerGl state asap : at the moment it means doing it in your `FetchInstanceConf` component but I'll probably move the logic somewhere else after...
- I used KeplerGl action `receiveMapConfig` to load the kepler config, did you try using this action in the past ? Only drawback I see is we must call it before loading any data, but sounds fine by me.

**Comment by lutangar** [comment]
@OlivPi  @Pipazoul feel free to make some comments on my WIP :
https://github.com/datatlas-erasme/front/pull/97/files

**Comment by Pipazoul** [comment]
@lutangar  Can you rephrase the sentence about the app reducer ? wich file is involved ? i'm not sure that i got it right. 

I dont remember using `receiveMapConfig `  before

**Comment by lutangar** [comment]
> the app reducer was sometimes populated with stuff but none of these was actually used, right ? I didn't see any selector trying to get something from the state.app slice...

@Pipazoul https://github.com/datatlas-erasme/front/blob/dev/src/store/reducer.js

Actions may be fired in the code (see https://github.com/datatlas-erasme/front/blob/dev/src/factories/panel-toggle.js#L37).
While it does populate `state.app`, there are no selectors (`useSelector()`) querying this part of the state : this  reducer is never used.

**Comment by Pipazoul** [comment]
> 



> > the app reducer was sometimes populated with stuff but none of these was actually used, right ? I didn't see any selector trying to get something from the state.app slice...
> 
> @Pipazoul https://github.com/datatlas-erasme/front/blob/dev/src/store/reducer.js
> 
> Actions may be fired in the code (see https://github.com/datatlas-erasme/front/blob/dev/src/factories/panel-toggle.js#L37). While it does populate `state.app`, there are no selectors (`useSelector()`) querying this part of the state : this reducer is never used.

Ah yes I havent used it

**Review by Pipazoul — APPROVED** [review]

**Review by OlivPi — APPROVED** [review]
